### PR TITLE
Add babel-polyfill dependency

### PR DESCRIPTION
**Pull request by TheAlexLichter**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,6 +70,7 @@
     "@nuxtjs/opencollective": "^0.1.0",
     "@nuxtjs/youch": "^4.2.3",
     "babel-loader": "^8.0.0",
+    "babel-polyfill": "^6.26.0",
```
Comment (aldarund) [inline package.json]: There is already a @babel/polyfill in devDeps

**Comment by clarkdo** [comment]
Pls wait, this issue need to be look deeper.

File: package.json (hunk 1 above)
Comment (manniL) [reply package.json]: Whoops.

**Comment by manniL** [comment]
@clarkdo Alright. Will freeze the PR until you give the go :relaxed:

**Comment by clarkdo** [comment]
Closed by https://github.com/nuxt/nuxt.js/commit/3ca2f026744b5fe760ecfe047d32d24522cbe315

**Pull request closed without merging**